internal/pkg/tfconf/module.go

```diff
@@ -22,8 +25,7 @@ type Module struct {
 	Providers      []*Provider `json:"providers" yaml:"providers"`
 	RequiredInputs []*Input    `json:"-" yaml:"-"`
 	OptionalInputs []*Input    `json:"-" yaml:"-"`
-
-	options *Options
+	Options        *Options    `json:"-" yaml:"-"`
```

Comment:
Ok I see why you had to make `Options` here exported, we shouldn't do that, because `Module` will eventually published publicly and consumer shouldn't have access to it (look #195 for detail if interested). It was my bad, I originally suggested to completely remove `OutputValues bool` from `print.Settings` which we should move it back.

And to be clear we need `OutputValues bool` in both `print.Settings` and `tfconf.Options`, because I'm trying to put a boundaries between the two in #195 . I mean `tfconf` package must not have dependency to `print` package.